swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -115,7 +134,7 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
         try String(reason.rawValue).write(
           to: SharedAccess.providerStopReasonURL, atomically: true, encoding: .utf8)
       } catch {
-        Log.error(
+        Log.warning(
```

Comment:
Demoted to warning since we don't rely on this for anything critically important.

Reply:
Well the entire idea of `SharedAccess` is to provide folder paths that are writable, so this will fail for log writing too if that were the case.

This is to handle the case the drive is full mainly or some weird edge case where a user mounted this dir to a removable disk or something then ejected it.

I can make it a bonafide error.